python/ray/air/tests/test_tracebacks.py

```diff
@@ -52,18 +52,16 @@ def failing(config):
         raise RuntimeError("Error")
 
     tuner = Tuner(failing)
-    results = tuner.fit()
-    assert len(str(results[0].error).split("\n")) <= 10
```

Comment:
Oh I think the point of these tests is to verify the traceback isn't super long....

Reply:
Oh after we changed to new persistent mode, the execution has been changed. So are we still need to have this test to check the number of lines in the stack trace?

Reply:
I _think_ thep urpose of this test is to make sure the trace isn't extremely long, what's the length of it now?

Reply:
I remember the length of this stack trace is around 13 lines.

Reply:
We should probably just bump this for now, and see if there's a way to clean up the stacktrace as a follow-up.

Reply:
Yeah sure, I can add this check back. 

I removed this test because verifying the precise number of lines in the stack trace seems unnecessary. Adjusting the internal code path shouldn't result in failing a test that's unrelated.

Reply:
Yeah, but again I think the whole point of this test is to ensure that our stacktraces aren't super long (because of internal code paths).